tools/roslaunch/test/unit/test_roslaunch_rlutil.py

```diff
@@ -89,3 +89,13 @@ def test_roslaunch_check_pass_all_args(self):
         filename = os.path.join(get_example_path(), 'example-pass_all_args.launch')
         error_msg = roslaunch.rlutil.check_roslaunch(filename)
         assert error_msg is None
+        
+    def test_check_roslaunch(self):
+        from roslaunch.rlutil import check_roslaunch
+
+        test_ignore_unset_args_p = os.path.join(get_test_path(), 'test', 'xml', 'test-ignore-unset-args.launch')
+        try:
+            result = check_roslaunch(test_ignore_unset_args_p, ignore_unset_args=True)
+            self.assertEqual(result, None)
+        except roslaunch.RLException:
+            pass
```

Comment:
Don't we want the test to fail when an exception is caught here?

Reply:
You're probably right. 

Not sure why I put that in there in the first place but it might've been to ignore specific read/write errors from check_roslaunch(). In any case, passing over any instance of RLException is probably a bad idea.